inventory/src/main/resources/application.properties

```diff
@@ -3,6 +3,6 @@ spring.application.name=inventory
 spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:mysql://localhost:3306/inventoryDb?createDatabaseIfNotExist=true
 spring.datasource.username=root
-spring.datasource.password=Philstar@123
```

Comment:
Always include database credentials in an .env file. Do not enter those things directly.